Jenkinsfile

```
@@ -79,6 +79,99 @@ pipeline {
                             'CAUTION: only use in combination with a reduced ' +
                             'number of tests specified with the TestTag ' +
                             'parameter.')
+        booleanParam(name: 'CI_BUILD_PACKAGES_ONLY',
+                     defaultValue: false,
+                     description: 'Only build RPM and DEB packages')
+        string(name: 'CI_RPM_TEST_VERSION',
+               defaultValue: '',
+               description: 'Package version to use instead of building. example: 1.3.103-1, 1.2-2')
+        string(name: 'CI_HARDWARE_DISTRO',
+               defaultValue: 'centos7',
+               description: 'Distribution to use for CI Hardware Tests')
+        string(name: 'CI_CENTOS7_TARGET',
+               defaultValue: 'el7',
+               description: 'Image to used for Centos 7 CI tests')
+        string(name: 'CI_CENTOS8_TARGET',
+               defaultValue: 'el8.3',
+               description: 'Image to used for Centos 8 CI tests')
+        string(name: 'CI_LEAP15_TARGET',
+               defaultValue: 'leap15.2',
+               description: 'Image to use for OpenSUSE Leap CI tests')
+        string(name: 'CI_UBUNTU20.04_TARGET',
+               defaultValue: 'ubuntu20.04',
+               description: 'Image to used for Ubuntu 20 CI tests')
+        booleanParam(name: 'CI_RPM_el7_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 7')
+        booleanParam(name: 'CI_RPM_el8_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 8')
+        booleanParam(name: 'CI_RPM_leap15_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for Leap 15')
+        booleanParam(name: 'CI_DEB_Ubuntu20_BUILD',
+                     defaultValue: false,
+                     description: 'Do not build DEB packages for Ubuntu 20')
```

NOBUILD to be similar to other params?
```suggestion
        booleanParam(name: 'CI_DEB_Ubuntu20_NOBUILD',
                     defaultValue: false,
                     description: 'Do not build DEB packages for Ubuntu 20')
```